File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/user/exception/ForbiddenUserException.java
package band.gosrock.domain.domains.user.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class ForbiddenUserException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new ForbiddenUserException();

    private ForbiddenUserException() {
        super(UserErrorCode.USER_FORBIDDEN);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/user/exception/ForbiddenUserException.kt
package band.gosrock.domain.domains.user.exception

import band.gosrock.common.exception.DuDoongCodeException

class ForbiddenUserException private constructor() : DuDoongCodeException(UserErrorCode.USER_FORBIDDEN) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = ForbiddenUserException()
    }
}
